### ISSUE TITLE
Use #variable for extension extensionality

### ISSUE THREAD

**Issue by fizruk** [emilyriehl/yoneda#26]
This is only a minor suggestion, I'm happy to take feedback.

Currently, we use `(extext : ExtExt)` parameter in many definitions.
However, we could instead use [`#variable` and `uses (...)` notation](https://rzk-lang.github.io/rzk/v0.5.2/reference/sections.rzk/) to write as follows:

```rzk
#lang rzk-1

-- Some definitions in this file rely on extension extensionality assumption
#variable extext : ExtExt

...

#def eq-ext-htpy uses (extext)
  ( I : CUBE)
  ( ψ : I -> TOPE)
  ( ϕ : ψ -> TOPE)
  ( A : ψ -> U)
  ( a : (t : ϕ) -> A t)
  ( f g : {t : I | ψ t} -> A t [ ϕ t |-> a t ])
  : ({t : I | ψ t} -> (f t = g t) [ ϕ t |-> refl ]) -> (f = g)
  := first (first (extext I ψ ϕ A a f g))
```

The benefit is that we do not need to pass `extext` explicitly and I think `uses (extext)` highlights this parameter as an assumption and distinguishes it from the regular parameters.

Note that `rzk` will issue an error if `extext` is used implicitly.

**Comment by emilyriehl** [comment]
This seems like an excellent suggestion to me. Should we treat funext similarly?

**Comment by fizruk** [comment]
Yes, I would treat all axioms this way, via global `#variable` declarations.

**Comment by fizruk** [comment]
Please, note that `#assume` is an alias for `#variables`. Perhaps, `#assume` is better for documentation of axioms.
